src/python/CRABClient/JobType/Analysis.py

```diff
@@ -156,7 +156,7 @@ def run(self, filecacheurl = None):
                     msg += " When Data.lumiMask is not specified, Data.runRange can not include more than 50000 runs."
                     raise ConfigurationException(msg)
                 lumi_list = LumiList(runs = run_list)
-        if lumi_list:
+        if lumi_list != None:
```

Comment:
Well, it seems to me at this point the client knows nothing useful will get done. It shouldn't need to round trip to the server to say "Hey, there is no work to do!"